Docs: SigV4 What's New and AWS Elasticsearch documentation (#28506)
grafana/grafana: docs/sources/whatsnew/whats-new-in-v7-3.md

This topic includes the release notes for Grafana v7.3. For all details, read the full [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).

The main highlights are:

- [**Google Cloud Monitoring:** Out of the box dashboards]({{< relref "#cloud-monitoring-out-of-the-box-dashboards" >}})
- [**Shorten URL for dashboards and Explore**]({{< relref "#shorten-url-for-dashboards-and-explore" >}})

> - [**SigV4 Authentication for Amazon Elasticsearch Service**]({{< relref "#sigv4-authentication-for-AWS-users" >}})

Reviewer: ```suggestion
- [**SigV4 Authentication for Amazon Elasticsearch Service**]({{< relref "#sigv4-authentication-for-aws-users" >}})
```

> ## Cloud monitoring out-of-the-box dashboards

Reviewer 2: ```suggestion
## Google Cloud monitoring out-of-the-box dashboards
```

> The updated Cloud monitoring data source is shipped with pre-configured dashboards for five of the most popular GCP services:

Reviewer 2: Please specify Google Cloud or Grafana Cloud to avoid confusion. @richardqlam and @matthewhelmke , something to watch for.

Reviewer 2: Spell out what GCP stands for on the first usage.

> This is an amazing new feature that was created in cooperation with one of our community members. The new **share shortened link** capability allows you to create smaller and simpler URLs of the format `/goto/:uid` instead of using longer URLs that can contain complex query parameters. In Explore, you can create a shortened link by clicking on the share button in Explore toolbar. In the dashboards, a shortened url option is available through the share panel or dashboard button.

Reviewer 2: Why is "share shortened link" bold? Only UI items should be bold.

## SigV4 authentication for AWS users

> You can now configure your Elasticsearch Data Source to access your Amazon Elasticsearch Service domain directly from Grafana.

Reviewer 3: ```suggestion
You can now configure your Elasticsearch data source to access your Amazon Elasticsearch Service domain directly from Grafana.
```

> For more details, see the [Elasticsearch docs]({{<relref "../datasources/elasticsearch/#aws-signature-version-4-authentication">}})

Reviewer 2: ```suggestion
For more details, refer to the [Elasticsearch docs]({{<relref "../datasources/elasticsearch/#aws-signature-version-4-authentication">}}).
```
